LLama.SemanticKernel/LLamaSharpPromptExecutionSettings.cs

```diff
@@ -1,10 +1,22 @@
-﻿using Microsoft.SemanticKernel;
+﻿
+/* Unmerged change from project 'LLamaSharp.SemanticKernel (netstandard2.0)'
+Before:
+using Microsoft.SemanticKernel;
+After:
+using LLamaSharp;
+using LLamaSharp.SemanticKernel;
+using LLamaSharp.SemanticKernel;
+using LLamaSharp.SemanticKernel.ChatCompletion;
+using Microsoft.SemanticKernel;
+*/
+using LLamaSharp.SemanticKernel.ChatCompletion;
+using Microsoft.SemanticKernel;
 using System.Text.Json;
 using System.Text.Json.Serialization;
 
-namespace LLamaSharp.SemanticKernel.ChatCompletion;
+namespace LLamaSharp.SemanticKernel;
 
-public class ChatRequestSettings : PromptExecutionSettings
+public class LLamaSharpPromptExecutionSettings : PromptExecutionSettings
```

Comment:
This is a reasonable renaming but introduces a break change. Could you please keep the `ChatRequestSettings` but mark it as `Obsolete`? It'll be better to completely remove it in the next 1~2 releases.

Reply:
OK!